docs/UserGuide.md

```diff
@@ -225,3 +240,9 @@ Action | Format, Examples
 **Find** | `find_session KEYWORD`<br><br>e.g. `find_session John`
 **Add** | `add_session n/STUDENT_NAME d/DATE t/TIME k/DURATION s/SUBJECT f/FEE`<br><br> e.g. `add_session n/John Doe d/2021-01-01 t/18:00 k/120 s/Biology f/80`
 **Delete** | `delete_session n/STUDENT_NAME i/SESSION_INDEX`<br><br>e.g. `delete_session n/John Lee i/1`
+
+**Others**
+
+Action | Format, Examples
+--------|------------------
+**fee** | `fee n/STUDENT_NAME m/MONTH y/YEAR`<br><br>e.g. `fee n/John Lee m/1 y/2021`
```

Comment:
Then under action "Student fee by month", what do you think?

Reply:
Actually not sure if its ambiguous, I think if we really want to be specific "Student fee for a particular month and year" would be more accurate. Will update to this version first.